WebHostLib/templates/playerOptions/macros.html

```diff
@@ -196,7 +196,7 @@
 {% macro OptionTitle(option_name, option) %}
     <label for="{{ option_name }}">
         {{ option.display_name|default(option_name) }}:
-        <span class="interactive" data-tooltip="{% filter dedent %}{{(option.__doc__ | default("Please document me!"))|escape }}{% endfilter %}">(?)</span>
+        <span class="interactive" data-tooltip="{{(option.__doc__ | default("Please document me!"))|replace('\n    ', '\n')|escape|trim}}">(?)</span>
```

Comment:
This looks like it assumes PEP8 formatting. Can we be assured all worlds conform to PEP8, and if not, are we okay with the idea that worlds which don't conform to it might have borked tooltip text?

Reply:
What would it break on?

I think

1.
```
"""stuff on a single line"""
```
2.
```
    """First line not indented
    Second line indented
    """
```
3.
```
    """
    First line indented
    Second line indented
    """
```

All work with this.
But yeah I mean there could be something I'm missing.